public/js/analytics_tracking_protection.js

```diff
@@ -50,15 +50,15 @@ if(typeof(ga) !== "undefined") {
   ga("set", "anonymizeIp", true);
   ga("set", "transport", "beacon");
 
-
-  // Strip token and hash values from pings sent to GA
   const loc = document.location;
-  let pageValue = loc.pathname + loc.search;
+  let pageValue = loc.href;
+
+    // Replace token and hash values before sending URL to GA
   if (loc.search && (loc.search.includes("token=") || loc.search.includes("hash="))) {
-    pageValue = loc.href.replace(/token=\w+&?/, "").replace(/hash=\w+&?/, "");
+    pageValue = loc.href.replace(/\?token=[A-Za-z0-9_-]+/, "").replace(/&hash=[A-Za-z0-9_-]+/, "");
   }
+
   ga("set", "location", pageValue);
```

Comment:
@groovecoder not sure if this matters/relates to the current situation in the dashboard but it looks like before #1275 we were only explicitly setting the "page" and "location" values if `loc.search.includes("token=") || loc.search.includes("hash=")`. I'm wondering if we should move this back to inside the `if` block?

Reply:
I was afraid that only setting the `page` and `location` values in the `if` block was causing #1265. 😢 Or at least, moving them outside the `if` block seemed to fix the bug described in those STR.

But if we can perform the same STR on this branch and verify we're not regressing the fix for #1265, then yeah - I'm happy to move them back how they were.

Reply:
I performed a spot-check of the #1265 STR on this branch and I still see the `utm_*` params in the `location` and `page` values in the GA debugger. So this change does NOT regress #1265 . 👍